server/methods/core/orders.js

```diff
@@ -56,6 +56,30 @@ export function ordersInventoryAdjust(orderId) {
   });
 }
 
+export function orderQuantityAdjust(orderId, refundedItem) {
+  check(orderId, String);
+
+  if (!Reaction.hasPermission("orders")) {
+    throw new Meteor.Error(403, "Access Denied");
```

Comment:
Passing in a number for the first argument is deprecated. We should use `"access-denied"`.